sdk/formrecognizer/ai-form-recognizer/src/transforms.ts

```diff
@@ -224,6 +242,9 @@ export function toFormFieldFromFieldValueModel(
     case "phoneNumber":
       value = original.valuePhoneNumber;
       break;
+    case "selectionMark":
+      value = original.valueSelectionMark;
```

Comment:
the service had a bug (I don't think this has been fixed) where `valueSelecitonMark` is empty even though the `type` is a `selectionMark`. The service does return the value in the `text` property, so we decided to use that in the SDK while the service fixes the bug.
i.e. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue_internal.cs#L60